package.json

```diff
@@ -357,8 +358,10 @@
     "date-fns": "^2.15.0",
     "fuse.js": "^6.4.6",
     "jszip": "^3.2.2",
+    "numeral": "^2.0.6",
```

Comment:
`numeral` also seems unmaintained, but at least it has no dependencies. Still, see question about whether we can omit the bloat by using built-in JS formatting.